target/linux/mediatek/mt7623/base-files/lib/preinit/07_set_iface_mac

```diff
@@ -1,24 +1,20 @@
 #!/bin/sh
```

Comment:
The shebang line must be removed according to this commit https://github.com/openwrt/openwrt/commit/48c1fdd046f409fd5e28932f1d53343fa214fbc4